packages/tokens/lib/scss/_variables.scss

```diff
@@ -1,5 +1,4 @@
 
-
```

Comment:
There was 2 lines at the start of this file, but after the update there's now 1?